reframe/core/buildsystems.py

```diff
@@ -655,6 +655,35 @@ def emit_build_commands(self, environ):
         return prepare_cmd + [' '.join(configure_cmd), ' '.join(make_cmd)]
 
 
+class EasyBuild(BuildSystem):
+    easyconfigs = fields.TypedField(typ.List[str])
+    options = fields.TypedField(typ.List[str])
+
+    def __init__(self):
+        super().__init__()
+        self.easyconfigs = []
+        self.options = []
+        self._installpath = None
+
+    def emit_build_commands(self, environ):
+        _easyconfigs = ' '.join(self.easyconfigs)
+        _options = ' '.join(self.options)
+        return [f"eb {_easyconfigs} --installpath {self._installpath} "
+                f"--tmpdir {self._installpath} "
+                f"--sourcepath {self._installpath} "
```

Comment:
Shouldn't these be different paths?

Reply:
@boegel thanks for having a look! The idea was to put everything inside reframe's `stage` directory in the same way the other build systems do. When the test passes, everything build-related would go away. Here I want to override eb's system configuration (for us, what the EasyBuild module sets) and move everything inside the `stage` dir.

I had dedicated variables for `tmpdir` and `sourcepath` but then I saw that easybuild creates a directory for each of them given the  path that that's passed: `<install-path>/software`, a random name for the `<install-path>/<tmp-name>`, etc. Then I just give the install dir only and let easybuild create the rest within it.